ui/analyse/src/ctrl.ts

```diff
@@ -687,7 +687,7 @@ export default class AnalyseCtrl {
     this.startCeval();
     if (!this.ceval.enabled()) {
       this.threatMode(false);
-      if (this.practice) this.togglePractice();
+      if (this.practice) this.practice = undefined;
```

Comment:
I'm pretty sure `togglePractice` isn't really needed here since there's no situation where this function turns `practice` _back on_ (we're only in this branch if `ceval` is disabled)

Supporting this conspiracy theory is that used to just set `this.practice = null` before that code got factored out into `practiceToggle` (https://github.com/lichess-org/lila/commit/f59597897b5d1c7bbaff27ba2a18b426dafdba6a)